src/validations/metadata-schema.ts

```diff
@@ -18,4 +18,35 @@ export const metadata = conditionalValidation({
     const errors = validator.errors?.map(($) => '' + $.message) || []
     return validationFailed(`The metadata for this entity type (${deployment.entity.type}) is not valid.`, ...errors)
   }
-})
+}
+
+type ADR = {
+  number: number
+  timestamp: number
+}
+
+const ADRMetadataVersionTimelines: Record<EntityType, ADR[]> = {
+  emote: [{ number: 74, timestamp: ADR_74_TIMESTAMP }].sort((v1, v2) => v1.timestamp - v2.timestamp),
+  scene: [],
+  profile: [],
+  wearable: [],
+  store: []
+}
+
+const metadataVersionIsCorrectForTimestamp: Validation = {
```

Comment:
maybe add a comment here that this method only implements emotes? as the error may be confused in the future

Reply:
added